src/context/auth/auth-reducer.js

```diff
@@ -0,0 +1,19 @@
+export const LOGIN = "LOGIN";
+export const LOGOUT = "LOGOUT";
+export const SIGNUP = "SIGNUP";
+export const initialAuthState = {
+  user: null,
+  isLogedIn: false,
+};
+export const authReducer = (state, action) => {
+  const { type, payload } = action;
+  switch (type) {
+    case LOGIN:
```

Comment:
why does case login: has no return or break statement?

Reply:
Returning same for login and signup